src/livekit/openIDSFU.ts

```diff
@@ -52,26 +62,35 @@ export async function getSFUConfigWithOpenID(
       error instanceof Error ? error : new Error("Unknown error"),
     );
   }
-  logger.debug("Got openID token", openIdToken);
+  logger?.debug("Got openID token", openIdToken);
 
-  logger.info(`Trying to get JWT for focus ${serviceUrl}...`);
+  logger?.info(`Trying to get JWT for focus ${serviceUrl}...`);
   const args: [CallMembershipIdentityParts, string, string, IOpenIDToken] = [
     membership,
     serviceUrl,
     livekitRoomAlias,
     openIdToken,
   ];
-  if (matrix2jwt) {
+  try {
+    // we do not want to try the old endpoint, since we are not sending the new matrix2.0 sticky events (no hashed identity in the event)
+    if (forceOldJwtEndpoint) throw new Error("Force old jwt endpoint");
```

Comment:
I am not really a fan of using an exception as a conditional control flow. 
I'd prefer that this is formatted as a normal conditional